test/SpokePoolClient.ValidateFill.ts

```diff
@@ -242,21 +246,25 @@ describe("SpokePoolClient: Fill Validation", async function () {
       spokePoolClient1.latestBlockNumber,
       10
     );
-    const searchRange2 = await spokePoolClient1._getBlockRangeForDepositId(
-      2,
-      spokePool1DeploymentBlock,
-      spokePoolClient1.latestBlockNumber,
-      10
+
+    expect(searchRange1.high).to.be.greaterThanOrEqual(deposit1Block);
+    expect(searchRange1.low).to.be.lessThanOrEqual(deposit1Block);
+
+    // Searching for deposit ID 2 that doesn't exist yet should throw.
+    await assertPromiseError(
+      spokePoolClient1._getBlockRangeForDepositId(2, spokePool1DeploymentBlock, spokePoolClient1.latestBlockNumber, 10),
```

Comment:
Might be nice to grab the depositId emitted when the deposit was made - i.e. up around line 213, where `deposit1Block` is extracted.

Reply:
Could you expand on this? My understanding is that this call is meant to throw